content/docs/buildpack-author-guide/create-buildpack/make-buildpack-configurable.md

```diff
@@ -134,13 +134,8 @@ You will notice that version of Ruby specified in the app's `.ruby-version` file
 [builder] ---> Downloading and extracting Ruby 2.5.0
 ```
 
-Congratulations! You've created your first configurable Cloud Native Buildpack that uses detection, image layers, and caching to create a runnable OCI image.
+Apart from the build logs, we can also store information about the dependencies that our buildpack has provided in the output image for introspection. We will do that in the next section.
```

Comment:
We could add a better follow up here? Not sure if this one is very exciting/correctly worded?

Reply:
Hmm. It definitely isn't super exciting, but the other page lead ins aren't super exciting either...
How about something like:
```suggestion
Next, let's see how buildpacks can store information about the dependencies provided in the output image for introspection. 
```